Umbraco-Cloud/Set-Up/Power-Tools/index.md

```diff
@@ -20,6 +20,27 @@ The power tools can be used for various things, we often refer to the tools in o
 * [Run an extraction manually](Manual-extractions)
 * [Generate UDA files](generating-uda-files)
 
+## Showing more than 199 items in Kudu
+ 
+Larger sites can often have more than 199 items in various folders, and you only have to view 199 files/folders by default.
+ 
+That can be increased by doing the following:
+ 
+1. Open the browser console while you're in Kudu.
+2. Type 
+```
+window.localStorage['maxViewItems'] = 999
+```
+Where 999 will be the new limit. You can, of course, set it to anything you like.
+
+3. Hit enter.
+4. Navigate out and back in to the folder you want to view the files in.
+5. You should now be able to view the folders/files up to the limit you've set it to.
```

Comment:
```suggestion
That can be increased by doing the following:
 
1. Open the browser console while you're in Kudu.
2. Type `window.localStorage['maxViewItems'] = 999` - Where 999 will be the new limit. This can be set to anything you like.
3. Hit enter.
4. Navigate out of and back into the folder you want to view the files in.
5. You should now be able to view the folders/files up to the limit you've set it to.
```